src/css/index.css

```diff
@@ -3,54 +3,63 @@
   --text-color: #202020;
   --button-color: #84347c;
   --button-color-hover: #6f6f6f;
-  --border-color: #c2c2c2; }
+  --border-color: #c2c2c2;
+}
 
 /******* Normalize *******/
 * {
   margin: 0;
   padding: 0;
-  box-sizing: border-box; }
+  box-sizing: border-box;
+}
 
 /****** Typography ******/
 h1 {
   font-family: "Open Sans", Arial, Helvetica, sans-serif;
-  font-size: 1.6rem; }
+  font-size: 1.6rem;
+}
 
 h2 {
-  font-size: 1.4rem; }
+  font-size: 1.4rem;
+}
 
 h3 {
-  font-size: 1.2rem; }
+  font-size: 1.2rem;
+}
 
 /* This is the main content */
 .fullContainer {
   display: grid;
   width: 100%;
   grid-template-columns: 1fr;
   height: auto !important;
-  width: 100% !important; }
+  width: 100% !important;
```

Comment:
Duplicate width property?

Reply:
there's a minus and an addition, but what's funny is that... I haven't modified it in so long so I'm not sure why it said I modified it. lol 
There's no duplicate here on my end. 

While we're on the topic!!  I think you noticed the `!important `part. I had to override a style that came with the map/or something else that I cannot control.  You may end up experiencing that too when playing around with the css @digilou . :(

Reply:
@PepperAddict I understand overrides, for sure. Usually I hate using `important`, but when using external resources, sometimes there is no other way.

Reply:
There's a package in the project called Milligram that I was using as a quick fix to create a layout while i was working on the sidebar. We should probably remove it since we're not using it. That's where the extra css is coming from.